src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -111,6 +111,18 @@
                                                                               Header="{Binding Path=PackageViewContextMenuUnmarkUninstallText}"
                                                                               ToolTip="{Binding Path=PackageViewContextMenuUnmarkUninstallTooltip}">
                                                             </MenuItem>
+                                                            <MenuItem Name="InstallButton"
+                                                                              Visibility="{Binding Path=CanInstall}"
```

Comment:
Looks like we have a lot of overlap with my PR for DYN-3726 (my PR gets smaller and smaller). I solved the visibility aspect of this in a slightly different way for the uninstall commands using a style that hides menu items that are disabled. Lets pick one of these solutions in the end.

Reply:
Sure....your solution sounds better
I can remove my changes

Reply:
You can leave it if you want. I can make that change in my PR as soon as this is merged.